File: Iniciante/URI 1037.ml
let valor = read_float();;
if (valor >= 0.0 && valor <= 25.0) then (
    Printf.printf "Intervalo [0,25]\n";
) else if (valor > 25.0 && valor <= 50.0) then (
    Printf.printf "Intervalo (25,50]\n";
) else if (valor > 50.0 && valor <= 75.0) then (
    Printf.printf "Intervalo (50,75]\n";
) else if (valor > 75.0 && valor <= 100.0) then (
    Printf.printf "Intervalo (75,100]\n";
) else (
    Printf.printf "Fora de intervalo\n";
)

File: Iniciante/URI 1037.rb
valor = gets.to_f
if valor >= 0 && valor <= 25
	puts "Intervalo [0,25]"
elsif valor > 25 && valor <= 50
	puts "Intervalo (25,50]"
elsif valor > 50 && valor <= 75
	puts "Intervalo (50,75]"
elsif valor > 75 && valor <= 100
	puts "Intervalo (75,100]"
else
	puts "Fora de intervalo"
end